srcs/runners_manager/vm_creation/openstack/OpenstackManager.py

```diff
@@ -13,6 +13,7 @@
 from runners_manager.monitoring.prometheus import metrics
 from runners_manager.runner.Runner import Runner
 from runners_manager.vm_creation.CloudManager import CloudManager
+from runners_manager.vm_creation.VmType import VmType
 from runners_manager.vm_creation.openstack.schema import OpenstackConfig
```

Comment:
Add `VmTypeConfig` so we can check the config settings and be shure it's match our needs